ase/api/src/buffer.c

```diff
@@ -532,3 +533,40 @@ fpga_result __FPGA_API__ ase_fpgaGetIOAddress(fpga_handle handle, uint64_t wsid,
 	}
 	return result;
 }
+
+/*
+ * A single PASID is associated with the process, independent of the number
+ * of FPGA ports connected or the open/close sequence.
+ */
+static int process_pasid;
+
+fpga_result __FPGA_API__ ase_fpgaBindSVA(fpga_handle handle, uint32_t *pasid)
+{
+	struct _fpga_handle *_handle = (struct _fpga_handle *)handle;
+	fpga_result result = FPGA_OK;
+	int err;
+
+	if (!pasid)
```

Comment:
The top-level doc allows pasid to be NULL.  Doc update or change the behavior here, I think.